linode/placementgroup/framework_models_unit_test.go

```diff
@@ -6,14 +6,17 @@ import (
 	"context"
 	"testing"
 
+	"github.com/hashicorp/terraform-plugin-framework/types"
 	"github.com/linode/linodego"
 	"github.com/stretchr/testify/require"
 )
 
 func TestFlattenPGModel(t *testing.T) {
+	label := "test-pgmodel"
+
 	pg := linodego.PlacementGroup{
 		ID:           123,
-		Label:        "test",
+		Label:        "test-pgmodel",
```

Comment:
```suggestion
		Label:        label,
```

Let's make `label` as the SSoT